darwin/main.m

```diff
@@ -243,3 +278,16 @@ void uiQueueMain(void (*f)(void *data), void *data)
 	// the signature of f matches dispatch_function_t
 	dispatch_async_f(dispatch_get_main_queue(), data, f);
 }
+
+void uiSize(uiControl *control, int *width, int *height)
+{
+	CGRect size = [[[sizeWrapper alloc] addControl:control] getSize];
+	*width = size.size.width;
+	*height = size.size.height;
+}
+
+void uiSetSize(uiControl *control, int width, int height)
+{
+	CGSize s = CGSizeMake(width, height);
+	[[[sizeWrapper alloc] addControl:control] setS:s];
+}
```

Comment:
Please use newlines at the end of files.

Reply:
Fixed.